### DISCUSSION TITLE
New FlaUI nuget release request

### DISCUSSION THREAD

**Discussion by alex-ngan** [FlaUI/FlaUI#471] [Q&A] [answered]
Hi,

I do not know if this is the correct place to ask but, I wonder when there is a new release available? We would like to update our FlaUI nuget package to include the changes, where some methods in the FlaUITestBase are able to be overriden thanks to the changes made by johan-eriksson.

Thanks in advance!

Best regards,
Alex

**Comment by Roemer** [comment, marked as answer]
That has just been done. Feel free to report new problems.
